app/assets/stylesheets/game.scss

```diff
@@ -1,21 +1,58 @@
 // Place all the styles related to the Game controller here.
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: http://sass-lang.com/
-.square{
+.block-form {
+  padding: 25px;
+  text-align: center;
+}
+
+.form {
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  text-align: center;
+}
+
+.grid {
+  margin-left: auto;
+  margin-right: auto;
+  text-align: center;
+}
+
+.square {
   width:100px;
   height:100px;
 }
 
-.v{
+.v {
   border-left:1px solid #000;
   border-right:1px solid #000;
 }
 
-.h{
+.h {
   border-top:1px solid #000;
   border-bottom:1px solid #000;
 }
 
-.turn_submit{
+.turn_submit {
   display: none;
 }
+
+.square_x:after {
+  content: "X";
+  font-size: xx-large;
+}
+
+.square_o:after {
+  content: "O";
+  font-size: xx-large;
+}
+
+.board-full {
+  pointer-events: none;
```

Comment:
Properties should be ordered -moz-filter, -ms-filter, -o-filter, -webkit-filter, filter, pointer-events